TTbarResLookUpTables.py

```diff
@@ -102,28 +102,25 @@ def multi_dict(K, type): # definition from https://www.geeksforgeeks.org/python-
 luts = multi_dict(2, str) #Annoying, but necessary definition of the dictionary
 
 def LoadDataLUTS(bdiscDirectory, Year, VFP, RemoveContam, ListOfLetters, uploadDir):
+    
     contam = ''
     if RemoveContam == True:
         contam = '_ttContaminationRemoved'
-    if Year != 0:
-        for letter in ListOfLetters:
-            for icat in list_of_cats:
-                df = pd.read_csv(uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + '_Data'+contam+'_' + icat[:4] + 'inc.csv')
-                luts['JetHT' + str(Year) + letter + '_Data'][icat] = df
-    else:
+    
+    for letter in ListOfLetters:
```

Comment:
There is a conflict in lines 109-123
```suggestion
    for letter in ListOfLetters:
            for icat in list_of_cats:
                csvfile = uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + '_Data'+contam+'_' + icat[:4] + 'inc.csv'
                if os.path.isfile(csvfile):
                    df = pd.read_csv(csvfile)
                else:
                    df = pd.read_csv(uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + letter + '_Data'+contam+'_' + icat +'.csv')
                luts['JetHT' + str(Year) + letter + '_Data'][icat] = df
```

Reply:
I didn't format the suggested change right, it should say to replace the current lines 109-123 with the added 110-117 lines here